packages/block-editor/src/components/block-removal-warning-modal/index.js

```diff
@@ -41,6 +41,19 @@ export function BlockRemovalWarningModal( { rules } ) {
 		return;
 	}
 
+	const message =
+		messageType === 'templates'
```

Comment:
It seems weird that a message mentioning "templates" which is a WP specific thing lives in the "block-editor" package? Can you clarify the reasoning?

Reply:
I was led astray by [the template wording that was already added previously](https://github.com/WordPress/gutenberg/pull/58138/files), and didn't make the connection that the block-editor should not be aware of templates. Will take another stab at this with this in mind.